install/setup/sql/mysql/kylo/0.9.0/update.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+MY_DIR=$(dirname $0)
+mysql -f -h $1 -u$2 --password=$3 < ${MY_DIR}/abandon-feed-jobs.sql
+echo "Updated to 0.9.0 release";
+
```

Comment:
Can you move this and the stored procedure code into a 0.8.0 folder instead of 0.9.0?
I know you made it 0.9.0 since our last release changed to 0.7.1 instead of 0.8.0.
This will now be going into our 0.8.0 release.